src/pods/canvas/components/relation/database-relation.component.tsx

```diff
@@ -46,6 +46,13 @@ const DatabaseRelationshipComponent: React.FC<DatabaseRelationshipProps> = ({
     isDrawLeftToRight
   );
 
+  const oneToOneRelationPath = `
+  M ${originXMinusFork} ${startCoords.y} 
+  H ${originXMinusFork - TABLE_CONST.HORIZONTAL_LEFT_EXTENSION} 
+  V ${endCoords.y} 
+  H ${originXMinusFork}
+  `;
```

Comment:
This is wrong, one to on relationships can be between two different tables, see screenshot:

![image](https://github.com/Lemoncode/mongo-modeler/assets/1457912/8dd51e47-8cfc-4433-873d-85ae24354dbd)

Please implement it in the way we mentioned, two separate components.

detect if is a recursive relations (fromTableId === toTableId)

Later one this nested relation will support one to many and many to one and one to one

Reply:
done